Lib/fontTools/misc/etree.py

```diff
@@ -208,7 +207,7 @@ def tostring(
     #   Char ::= #x9 | #xA | #xD | [#x20-#xD7FF] | [#xE000-#xFFFD] | [#x10000-#x10FFFF]
     # Here we reversed the pattern to match only the invalid characters.
     _invalid_xml_string = re.compile(
-        "[\u0000-\u0008\u000B-\u000C\u000E-\u001F\uD800-\uDFFF\uFFFE-\uFFFF]"
+        "[\u0000-\u0008\u000b-\u000c\u000e-\u001f\ud800-\udfff\ufffe-\uffff]"
```

Comment:
Is there away to disable this hex lower casing? I find `\udfff` to be less readable than `\uDFFF`.

Reply:
ugh.. looks like it's a new thing they added with https://github.com/psf/black/pull/2916

and black being black, you can't disable it :(

Reply:
ruff does the same so that's the canonical way we have to get used to i'm afraid
https://github.com/astral-sh/ruff/issues/13859